include/openssl/evperr.h

```diff
@@ -133,4 +133,9 @@
 # define EVP_R_XTS_DATA_UNIT_IS_TOO_LARGE                 191
 # define EVP_R_XTS_DUPLICATED_KEYS                        192
 
+/*
```

Comment:
This will get removed when evperr.h is regenerated. That's why i suggested putting it into cryptoerr_legacy.h. Just add it somewhere to the end - create a new section for this kind of legacy reason codes.

Reply:
I see, thanks! Moved to cryptoerr_legacy.h in the last commit.